inlong-sdk/dataproxy-sdk/src/main/java/org/apache/inlong/sdk/dataproxy/config/ProxyConfigManager.java

```diff
@@ -588,8 +588,12 @@ private ProxyConfigEntry getLocalProxyListFromFile(String filePath) throws Excep
         Map<String, Integer> streamIdMap = getStreamIdMap(localProxyAddrJson);
         proxyEntry.setGroupIdNumAndStreamIdNumMap(groupIdNum, streamIdMap);
         proxyEntry.setLoad(load);
-        if (localProxyAddrJson.has("cluster_id")) {
-            proxyEntry.setClusterId(localProxyAddrJson.get("cluster_id").getAsString());
+        if (localProxyAddrJson.has("data")) {
```

Comment:
The configuration of the cluster_id needs to be added to the local configuration file to deal with this problem. Are you trying to deal with the configuration pulled back from the manager? If this case, you need to modify the requestProxyList function.

Reply:
Good job. LGTM. It is suggested to be done in another pr.